### PR TITLE
Release 2.1.0

### PR DESCRIPTION
**Pull request by skrakau**
- add bin abundance estimation and generation of heatmap with abundances across samples
- publish Prodigal bin annotations obtained from BUSCO run
- template update
- update overview figure
- use assembly reproducibility setting for full-size test profile

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/mag/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by skrakau** [comment]
why is there a warning saying:
`nextflow_config - Config manifest.version should end in dev: '2.1.0'` ?

File: .nf-core.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Remove when not needed anymore!
```
Comment (d4straub) [inline .nf-core.yml:1]: just to make sure, this is still needed?

File: conf/test_hybrid_host_rm.config [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
-  min_length_unbinned_contigs = 1
-  max_unbinned_contigs = 2
-  busco_reference = "https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2020-03-06.tar.gz"
-  gtdb = false
+    // Input data
+    host_fasta                  = "https://raw.githubusercontent.com/nf-core/test-datasets/mag/host_reference/genome.hg38.chr21_10000bp_region.fa"
+    input                       = 'https://raw.githubusercontent.com/nf-core/test-datasets/mag/samplesheets/samplesheet.hybrid_host_rm.csv'
+    min_length_unbinned_contigs = 1
+    max_unbinned_contigs        = 2
+    busco_reference             = "https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2020-03-06.tar.gz"
```
Comment (d4straub) [inline conf/test_hybrid_host_rm.config:27]: I think you wrote that earlier, but why again `https://github.com/nf-core/test-datasets/raw/` to `https://raw.githubusercontent.com/nf-core/test-datasets` ?

**Review by d4straub — APPROVED** [review]
Nice!

_Edit:_

> why is there a warning saying:
nextflow_config - Config manifest.version should end in dev: '2.1.0' ?

_Because its the dev branch? no worries!_

File: .nf-core.yml (hunk 1 above)
Comment (skrakau) [reply .nf-core.yml:1]: it is

File: conf/test_hybrid_host_rm.config (hunk 2 above)
Comment (skrakau) [reply conf/test_hybrid_host_rm.config:27]: The template does it like this and I remembered someone saying this should be more robust or so

**Review by ggabernet — APPROVED** [review]
Looks good to me!

**Comment by ggabernet** [comment]
> why is there a warning saying:
> `nextflow_config - Config manifest.version should end in dev: '2.1.0'` ?

Yeah I think this is actually a bug, it should not give out this warning for PRs from dev to master

**Comment by skrakau** [comment]
Thanks @d4straub and @ggabernet !

**Pull request merged**